### PR TITLE
Change attach to show user name. Use connectionProfile.title to displ…

### PR DESCRIPTION
**Pull request by YurongHe**
Change attach to show user name. Use connectionProfile.title to display the same info as OE server node

Look like this:
image

### CODE REVIEW

**Comment by Charles-Gagnon** [comment]
That's getting pretty verbose - is there a max width? Make sure you're testing with really long DB names that you're connecting to.

Is this going to also update the values in the dropdown when expanded?

I'd also consider just having it show it as it is now but then have the mouseover tooltip contain the full name.

**Comment by YurongHe** [comment]
> That's getting pretty verbose - is there a max width? Make sure you're testing with really long DB names that you're connecting to.
> 
> Is this going to also update the values in the dropdown when expanded?
> 
> I'd also consider just having it show it as it is now but then have the mouseover tooltip contain the full name.

@Charles-Gagnon agreed. we discussed this with PM before to have max size. We haven't agree on this. Also we don't have show tip for SelectBox.

**Comment by YurongHe** [comment]
We had work item to track this.

**Comment by YurongHe** [comment]
> > That's getting pretty verbose - is there a max width? Make sure you're testing with really long DB names that you're connecting to.
> > Is this going to also update the values in the dropdown when expanded?
> > I'd also consider just having it show it as it is now but then have the mouseover tooltip contain the full name.
> 
> @Charles-Gagnon agreed. we discussed this with PM before to have max size. We haven't agree on this. Also we don't have show tip for SelectBox.

We have work item to track the issue.

**Comment by YurongHe** [comment]
> @YurongHe can you make sure that this works in a number of cases, like:
> 
> * When a friendly name is set for a server
> * When multiple connections show up in the attach to
> 
> My gut tells me that we have some code that parses the attach to value to determine the server name. Just want to make sure that that still works correctly

@chlafreniere this is fixed. but we noticed SelectBox doesn't fire select event occasionally.

File: src/sql/parts/notebook/models/notebookModel.ts [hunk 1]
```diff
@@ -637,34 +637,39 @@ export class NotebookModel extends Disposable implements INotebookModel {
...
+							//Selected a wrong connection, Attach to should be defaulted with 'Select connection'
+							this._onValidConnectionSelected.fire(false);
 						}
-						//Selected a wrong connection, Attach to should be defaulted with 'Select connection'
-						this._onValidConnectionSelected.fire(false);
-					}
-				});
+					});
+			} else {
+				throw new Error('No valid connection');
```
Comment (chlafreniere) [inline src/sql/parts/notebook/models/notebookModel.ts]: should we be calling fire(false) here as well?

**Pull request merged**